### PR TITLE
Account Profile Editing

### LINKED ISSUE #61: Editing an account profile
image

### PR DESCRIPTION
**Pull request by n-s-mordvinov**

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -138,6 +142,31 @@
...
+    "edit": {
+      "fields": {
+        "operatingName": {
+          "label": "Company name"
+        },
+        "avatar": {
+          "label": "Company avatar"
+        },
+        "address": {
+          "label": "Company adress"
```
[the remark this answers is not in the document]
Comment (sernik73) [reply src/locales/en/translation.json:155]: I split by page or component

File: src/form-helpers/account-edit/schema.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import * as yup from 'yup';
+
+export const schema = yup.object().shape({
+  address: yup.string().required('Required').default(''),
+  operatingName: yup.string().required('Required').default(''),
+  name: yup.string().required('Required').default(''),
+  email: yup.string().email().test('or', 'or', function validate(value) {
```
[the remark this answers is not in the document]
Comment (sernik73) [reply src/form-helpers/account-edit/schema.js:7]: done

File: src/pages/AccountProfile/index.jsx [hunk 3]
```diff
@@ -61,6 +74,13 @@ export const AccountProfile = () => {
           </Profile>
         )
       }
+      <Success
+        visible={ visibleSuccess }
+        duration="3000"
```
[the remark this answers is not in the document]
Comment (sernik73) [reply src/pages/AccountProfile/index.jsx]: done

File: src/locales/en/translation.json [hunk 4]
```diff
@@ -138,6 +142,31 @@
...
+        "email": {
+          "label": "Company E-mail"
+        },
+        "phone": {
+          "label": "Company phone"
+        }
+      }
+    },
+    "success": {
+      "message": "Changes saved"
```
[the remark this answers is not in the document]
Comment (sernik73) [reply src/locales/en/translation.json]: done

File: src/locales/en/translation.json [hunk 5]
```diff
@@ -138,6 +142,31 @@
     "ProfileContentBottom": {
       "leftLink": "All facilities",
       "rightLink": "All members"
+    },
+    "edit": {
+      "fields": {
+        "operatingName": {
+          "label": "Company name"
```
[the remark this answers is not in the document]
Comment (sernik73) [reply src/locales/en/translation.json]: done

File: src/locales/en/translation.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (sernik73) [reply src/locales/en/translation.json]: done

File: src/locales/en/translation.json [hunk 6]
```diff
@@ -138,6 +142,31 @@
...
+          "label": "Company adress"
+        },
+        "name": {
+          "label": "Legal entity"
+        },
+        "email": {
+          "label": "Company E-mail"
+        },
+        "phone": {
+          "label": "Company phone"
```
[the remark this answers is not in the document]
Comment (sernik73) [reply src/locales/en/translation.json]: done

File: src/locales/en/translation.json [hunk 7]
```diff
@@ -138,6 +142,31 @@
...
+          "label": "Company avatar"
+        },
+        "address": {
+          "label": "Company adress"
+        },
+        "name": {
+          "label": "Legal entity"
+        },
+        "email": {
+          "label": "Company E-mail"
```
[the remark this answers is not in the document]
Comment (sernik73) [reply src/locales/en/translation.json]: done

**Review by kirillta — APPROVED** [review]

**Pull request merged**